README.md

```diff
@@ -72,23 +72,32 @@ refer to the [online documentation](https://documentation.centreon.com/docs/cent
 
 First of all, check if you have these packages installed (Note that packages names come from Centos 7 distribution, so if some packages names don't match on your distribution try to find their equivalent names) : 
 	
-    git, make, cmake, gcc-c++, python3, python3-pip3, lua-devel, rrdtool-devel, gnutls-devel.
+    git, make, cmake3, gcc-c++, python3, python3-pip3, lua-devel, rrdtool-devel, gnutls-devel, mariadb-devel, perl-ExtUtils-Embed.
 
 If they are not installed, please install them.
 
+For the projet compilation you need to have conan installed. To install conan you need pip3 if you are using python3 (python package manager). You can install conan like that.
+
+    $> pip3 install conan
+
+You will need to use cmake3, if you already had cmake just do those commands to use cmake3 instead of cmake by renaming it like so :
+
+    $> mv /usr/bin/cmake /usr/bin/cmake.old
+    $> ln -s /usr/bin/cmake3 /usr/bin/cmake
```

Comment:
This is true only on a centos7 linux. Others distributions provide cmake 3 by default and its executable is already cmake